castle/cms/tiles/video.py

```diff
@@ -41,17 +43,24 @@ def loop(self):
     def tweak_video_url(self, url):
         # add params for properties...
         parts = url.split('/')
+        
+        # gets the 't=' start time value before cleaning the url
+        parsed = urlparse.urlparse(url)
+        start_time = urlparse.parse_qs(parsed.query)['t'][0]
```

Comment:
If there is no t value in the url, this will likely cause a KeyError.  So we'll want to add a try/except block or an if statement to make sure 't' is there.  Otherwise we can merge this one in soon!